keyboards/gizmo_engineering/gk6/keymaps/12x5/keymap.c

```diff
@@ -0,0 +1,261 @@
+ * |Debug |      |      |      |      |      |      |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_RAISE] = LAYOUT_gk6_5x12( \
+  _______, _______,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, _______,  _______, _______, _______, _______, _______, KC_UNDS, KC_PLUS, KC_LCBR, KC_RCBR, _______, \
+  _______, _______,  _______, _______, _______, _______, _______, KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, _______, \
+  _______, RGB_TOG,  RGB_MOD, RGB_HUI, RGB_SAI, RGB_VAI, RGB_SPI, _______, _______, _______, KC_PIPE, KC_BSLS, \
+  DEBUG,   _______,  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______  \
+),
+
+/* Lower
+ * ,-----------------------------------------------------------------------------------.
+ * |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  | F7   |  F8  |  F9  |  F10 |  F11 |  F12 |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | CAPS | PSCR |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      | Home | End  | Ins  | Play |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |  <<  |  >>  | Mute | Vol- | Vol+ |      |      |      |      | PgUp |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      | PgDn |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_LOWER] = LAYOUT_gk6_5x12( \
+  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  \
+  KC_CAPS, KC_PSCR, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+  _______, KC_HOME, KC_END,  KC_INS,  KC_MPLY, _______, _______, _______, _______, _______, _______, _______, \
+  _______, KC_MPRV, KC_MNXT, KC_MUTE, KC_VOLD, KC_VOLU, _______, _______, _______, _______, KC_PGUP, _______, \
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_PGDN, _______  \
+),
+
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+        case LOWER:
+          if (record->event.pressed) {
+            layer_on(_LOWER);
+          } else {
+            layer_off(_LOWER);
+          }
+          return false;
+          break;
+        case RAISE:
+          if (record->event.pressed) {
+            layer_on(_RAISE);
+          } else {
+            layer_off(_RAISE);
+          }
+          return false;
+          break;
+      }
+    return true;
+};
+
+const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
```

Comment:
This should go in the keyboard's c file, not here

Reply:
Good one @drashna, the reason I've put this in here is they are different for each layout, as we don't LEDs without switches to turn on.

Reply:
Yeah, it should be in the keyboard's C file, no matter what. 

You could use `rgb_matrix_indicator_user` to effectively disable the LEDs in question. 
Or create revisions/subfolders for the different layouts.